UnitTests/GitUI.Tests/SplitterManagerTest.cs

```diff
@@ -6,216 +6,215 @@
+
+                // assert
+                splitter.SplitterDistance.Should().Be(splitterDistance * 2);
+            }
+        }
+
+        [TestCase(100)]
+        [TestCase(-100)]
+        public void ForFixedPanel1_WhenWidthChanges_DistanceDoesNotChange(int deltaWidth)
+        {
+            // arrange
+            const string splitterName = "splitterName";
+            const int splitterWidth = 200;
+            const int splitterDistance = 70;
+            {
+                var splitManager = new SplitterManager(_settings);
+                SplitContainer splitter = CreateVerticalSplitContainer();
+                splitter.Width = splitterWidth;
+                splitter.SplitterDistance = splitterDistance;
+                splitManager.AddSplitter(splitter, splitterName);
+                splitManager.SaveSplitters();
+            }
+
+            {
+                // act
+                var splitManager = new SplitterManager(_settings);
+                SplitContainer splitter = CreateVerticalSplitContainer();
+                splitManager.AddSplitter(splitter, splitterName);
+                splitter.Width = splitterWidth + deltaWidth;
+                splitter.FixedPanel = FixedPanel.Panel1;
+                splitManager.RestoreSplitters();
+
+                // assert
+                splitter.SplitterDistance.Should().Be(splitterDistance);
+            }
+        }
+
+        [TestCase(100)]
+        [TestCase(-100)]
+        public void ForFixedPanel2_WhenWidthChanges_DistanceChangesAlong(int deltaWidth)
+        {
+            // arrange
+            const string splitterName = "splitterName";
+            const int splitterWidth = 200;
+            const int splitterDistance = 130;
+            {
+                var splitManager = new SplitterManager(_settings);
+                SplitContainer splitter = CreateVerticalSplitContainer();
+                splitter.Width = splitterWidth;
+                splitter.SplitterDistance = splitterDistance;
+                splitManager.AddSplitter(splitter, splitterName);
+                splitManager.SaveSplitters();
+            }
+
+            {
+                // act
+                var splitManager = new SplitterManager(_settings);
+                SplitContainer splitter = CreateVerticalSplitContainer();
+                splitManager.AddSplitter(splitter, splitterName);
+                int splitterNewWidth = splitterWidth + deltaWidth;
+                splitter.Width = splitterNewWidth;
+                splitter.FixedPanel = FixedPanel.Panel2;
+                splitManager.RestoreSplitters();
+
+                // assert splitter moved by the width delta
+                // Note: if the width of the splitter control itself is not regarded,
+                // this splitter control would move to the left every time the splitter container is restored.
+                splitter.SplitterDistance.Should().Be(splitterDistance + deltaWidth + splitter.SplitterWidth - 2);
```

Comment:
`-2` here

Reply:
I did it but it didn't help.
The situation is that nested elements affect the sizes in a cascade.